awx/ui/src/screens/Setting/shared/data.allSettingOptions.json

```diff
@@ -3706,6 +3706,28 @@
           "required": true,
           "read_only": false
         }
+      },
+      "SOCIAL_AUTH_SAML_USER_FLAGS_BY_ATTR": {
+        "type": "nested object",
+        "required": false,
+        "label": "SAML User Flags Attribute Mapping",
+        "help_text": "Used to map super users and system auditors from SAML.",
+        "category": "SAML",
+        "category_slug": "saml",
+        "placeholder": {
+          "is_superuser_attr": "saml_attr",
+          "is_superuser_value": "value",
+          "is_superuser_role": "saml_role",
+          "is_system_auditor_attr": "saml_attr",
+          "is_system_auditor_value": "value",
+          "is_system_auditor_role": "saml_role"
+        },
+        "default": {},
+        "child": {
+          "type": "field",
+          "required": true,
+          "read_only": false
+        }
```

Comment:
I'm adding @nixocio as a reviewer to this PR because you're touching some of the configure-tower-in-tower UI code here.

Do you even need the "placeholder" if it can come from the API options?

Reply:
This data is just used on tests. You are right values come from the API options.